### PR TITLE
Slither Warning

### PR DESCRIPTION
**Pull request by AlbyGiaco**

### CODE REVIEW

File: contracts/PositionManager.sol [hunk 1]
```diff
@@ -96,17 +96,39 @@ contract PositionManager is IPositionManager, ERC721Holder {
...
         StorageStruct storage Storage = PositionManagerStorage.getStorage();
         Storage.owner = _owner;
         Storage.uniswapAddressHolder = IUniswapAddressHolder(_uniswapAddressHolder);
         Storage.registry = IRegistry(_registry);
         Storage.aaveAddressHolder = IAaveAddressHolder(_aaveAddressHolder);
     }
 
+    ///@notice withdraw uniswap position NFT from the position manager
+    ///@param tokenId ID of withdrawn token
+    function withdrawUniNft(uint256 tokenId) external override onlyOwner {
```
Comment (RiccardoGalbusera) [inline contracts/PositionManager.sol:109]: We decided to remove this function, maybe pull from main to update this change!

File: contracts/PositionManager.sol [hunk 2]
```diff
@@ -96,17 +96,39 @@ contract PositionManager is IPositionManager, ERC721Holder {
...
+            tokenId,
+            '0x0'
+        );
+        emit WithdrawUni(msg.sender, tokenId);
+    }
+
     ///@notice remove awareness of tokenId UniswapV3 NFT
     ///@param tokenId ID of the NFT to remove
-    function removePositionId(uint256 tokenId) public override {
+    function removePositionId(uint256 tokenId) external override {
```
Comment (RiccardoGalbusera) [inline contracts/PositionManager.sol:131]: Does this change keep tests running? Because when we use delegate call, position manager may call this function

**Review by RiccardoGalbusera — CHANGES_REQUESTED** [review]

**Comment by RiccardoGalbusera** [comment]
Also, compiling on this branch return these errors, please fix them (I think that's because recent merge of ORB-396 changes)

image

File: contracts/PositionManager.sol (hunk 2 above)
Comment (AlbyGiaco) [reply contracts/PositionManager.sol:131]: Not for now, so better keep that external for now

**Review by nicolas-racchi — COMMENTED** [review]
Due to some confusion, some require messages throughout the contracts follow the "Contract::method: <Error name>" convention, while some of them use "Contract:method:: <Error name>"

**Pull request merged**